wger/gym/templates/gym/inactive_gym_users_list.html

```diff
@@ -0,0 +1,385 @@
+{% extends "base.html" %}
+{% load i18n staticfiles wger_extras django_bootstrap_breadcrumbs %}
+
+{% block title %}{{gym}}{% endblock %}
+
+{% block breadcrumbs %}
+    {{ block.super }}
+
+    {% if perms.gym.manage_gyms %}
+        {% breadcrumb "Gyms" "gym:gym:list" %}
+        {% breadcrumb_raw gym "gym:gym:user-list" gym.pk %}
+        {% breadcrumb_raw name "gym:gym:inactive-user-list" gym.pk %}
+    {% endif %}
+{% endblock %}
+
+{% block content %}
+{% if perms.gym.manage_gym or perms.gym.gym_trainer %}
+
+<link rel="stylesheet" type="text/css" href="{% static 'bower_components/datatables/media/css/dataTables.bootstrap.min.css' %}">
+<script src="{% static 'bower_components/datatables/media/js/jquery.dataTables.min.js' %}" ></script>
+<script src="{% static 'bower_components/datatables/media/js/dataTables.bootstrap.min.js' %}" ></script>
+<script>
+$(document).ready( function () {
+    /* Make table sortable */
+    $('#main_member_list').DataTable({
+        paging: false,
+        bFilter: true,
+        bInfo : false
+    });
+});
+</script>
+
+<table class="table table-hover" id="main_member_list">
+<thead>
+<tr>
+    {% for key in user_table.keys %}
+        <th>{{ key }}</th>
+    {% endfor %}
+
+</tr>
+</thead>
+<tbody>
+{% for current_user in user_table.users %}
+{% if not current_user.obj.is_active %}
+<tr>
+    <td>
+        {{current_user.obj.pk}}
+    </td>
+    <td>
+        <a href="{% url 'core:user:overview' current_user.obj.pk %}">{{current_user.obj}}</a>
+    </td>
+    <td>
+        {{current_user.obj.get_full_name}}
+    </td>
+    <td data-order="{{current_user.last_log|date:'U'}}">
+        {{current_user.last_log|default:'-/-'}}
+    </td>
+    {% if show_gym %}
+    <td>
+        {% if current_user.obj.userprofile.gym_id %}
+            <a href="{{ current_user.obj.userprofile.gym.get_absolute_url }}">
+            {{ current_user.obj.userprofile.gym }}
+            </a>
+        {% else %}
+            -/-
+        {% endif %}
+    </td>
+    {% endif %}
+         <td>
+            <a href="{{ current_user.obj.userprofile.gym.get_absolute_url }}">
+            </a>
+            Inactive
+    </td>
+</tr>
+{% endif %}
+{% endfor %}
+</tbody>
+</table>
+{% endif %}
+
+
+<h4>{% trans "Administrators and trainers" %}</h4>
+<table class="table table-hover">
+<thead>
+<tr>
+    <th style="width: 10%;">{% trans "ID" %}</th>
+    <th style="width: 40%;">{% trans "Username" %}</th>
+    <th>{% trans "Name" %}</th>
+    {% if perms.gym.manage_gym or perms.gym.manage_gyms %}
+        <th style="text-align: right;">{% trans "Roles" %}</th>
+    {% endif %}
+</tr>
+</thead>
+<tbody>
+{% for current_user in object_list.admins %}
+<tr>
+    <td>
+        {{current_user.obj.pk}}
+    </td>
+    <td>
+        {{current_user.obj}}
+
+        {% if current_user.perms.gym_trainer %}
```

Comment:
***Code Quality: Nitpick***
There is a whole lot of repition goin on here. Can he variable be passed from the views.py and just so that you just have one
```
 <span class="label label-primary">{% var %}</span>
```